irods/test/connection_test.py

```diff
@@ -105,22 +105,53 @@ def test_that_connection_timeout_works__issue_377(self):
             if local_file:
                 os.unlink(local_file.name)
 
-    def assert_timeout_value_propagated_to_socket(self, session, timeout_value):
+    def _assert_timeout_value_is_propagated_to_all_sockets__issue_569(self, session, timeout_value = 'POOL_TIMEOUT_SETTING'):
+        pool = session.pool
+        if timeout_value == 'POOL_TIMEOUT_SETTING':
+            timeout_value = pool.connection_timeout
```

Comment:
I see `timeout_value` is written to, but I don't see where it's being used by the sockets.

Reply:
Perhaps it would help to rename this to `expected_timeout_value`?

Reply:
In doing an irods operation, connections will be created in the session pool, then released into one of the two sets, pool.idle or active.  (usually idle, between  statements of application or test code).  We then assert that calling gettimeout on the sockets of the temporarily unused (idle ) connection gives us the correct value.

Reply:
> Perhaps it would help to rename this to `expected_timeout_value`?

Yes, probably.